src/_includes/layouts/component-preview.njk

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="{{ locale }}" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>{% block page_title %}{{ title }} - {{ base[locale].gcds }}{% endblock %}</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+
+      <script src="{{ '/scripts/component-preview-iframe.js' | url }}"></script>
+
+    <!-- Utility framework -->
+    <link rel="stylesheet" href="{{ '/gcds-utility.min.css' | url }}" />
+
+    <link rel="icon" type="image/x-icon" sizes="96x96" href="{{ '/favicon.ico' | url }}">
+
+    <!-- Icons Font Awesome (to access icons, import Font Awesome) -->
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.4.2/css/all.min.css" crossorigin="anonymous">
```

Comment:
Would it be possible to use `{{ links.fontAwesome }}` here? I also see it hard-coded within `layouts/base.njk` and `layouts/home.njk`.